modules/lounge.js

```diff
@@ -1,16 +1,23 @@
 "use strict";
 var commands = function(bot, options, action) {
-	let target = action.target;
-	if (action.target === bot.user.nick) {
-		target = action.nick;
-	}
+	const message = action.message;
+	const words = message.split(" ");
+	let target = null;
+	let args = words.slice(); // copy words
+	args.splice(0, 1); // remove command
+	args = args.join(" ");
+	target = args.split(/[ ,]/g).filter(Boolean).join(", ");
 
-	if (action.message === options.commandPrefix + "css") {
-		bot.say(target, "https://github.com/thelounge/lounge/wiki/CSS-Modifications");
-	} else if (action.message === options.commandPrefix + "rp") {
-		bot.say(target, "https://github.com/thelounge/lounge/wiki/Reverse-Proxies");
-	} else if (action.message === options.commandPrefix + "diff") {
-		bot.say(target, "https://github.com/thelounge/lounge/wiki/Differences-between-The-Lounge-and-Shout");
+	let url = "";
+	if (words[0] === options.commandPrefix + "css") {
```

Comment:
If you turn these commands into a map (command, reply aka url) then you can avoid a lot of the code duplication you have.